cmake/Modules/winconf-mingw.cmake

```diff
@@ -6,4 +6,4 @@ list(APPEND LibtorrentRasterbar_CUSTOM_DEFINITIONS
     -D_FILE_OFFSET_BITS=64
     -D__USE_W32_SOCKETS)
 
-link_libraries(advapi32 iphlpapi ole32 powrprof shell32 user32 wsock32 ws2_32)
```

Comment:
Was this causing any trouble? qbt do depend on that lib.

Reply:
No Now I directly add the required lib to the library itself so this is not needed anymore. This actually adds global link Library i.e every library or executable compiled will automatically link to this library but now it will only link to qbt_base

Reply:
Should i do it with link_libraries inside win-conf.msvc